src/olm/account/mod.rs

```diff
@@ -113,17 +115,17 @@ impl Account {
         }
     }
 
-    /// Get the IdentityKeys of this Account
+    /// Get the [`IdentityKeys`] of this Account
     pub const fn identity_keys(&self) -> IdentityKeys {
         IdentityKeys { ed25519: self.ed25519_key(), curve25519: self.curve25519_key() }
     }
 
-    /// Get a reference to the account's public Ed25519 key
+    /// Get a copy of the account's public Ed25519 key
     pub const fn ed25519_key(&self) -> Ed25519PublicKey {
         self.signing_key.public_key()
     }
 
-    /// Get a reference to the account's public Curve25519 key
+    /// Get a copy to the account's public Curve25519 key
```

Comment:
copy to -> copy of

Reply:
50b8fd3cbeb8bda10bf3cf0d19904fbe7fbc0b5d